src/theme/as24/colors.ts

```diff
@@ -2,7 +2,7 @@
 import { Colors } from '@chakra-ui/react';
 
 const colors: Colors = {
-  brand: {
+  auto: {
```

Comment:
The idea was to be able to use `brand` colour without needing to know which theme we're using.
When styling individual components we want to use universal tokens and have a theme take care of switching values for us.